test/functional/test-resources.js

```diff
@@ -994,23 +994,11 @@ describe('Resources changeSize', () => {
       vsyncSpy = sandbox.stub(resources.vsync_, 'run');
     });
 
-    it('should NOT change size and calls overflowCallback', () => {
-      resources.scheduleChangeSize_(resource1, 111, 222, false);
-      resources.mutateWork_();
-      expect(resources.requestsChangeSize_.length).to.equal(0);
-      expect(resource1.changeSize.callCount).to.equal(0);
-      expect(overflowCallbackSpy.callCount).to.equal(1);
-      expect(overflowCallbackSpy.firstCall.args[0]).to.equal(true);
-      expect(overflowCallbackSpy.firstCall.args[1]).to.equal(111);
-      expect(overflowCallbackSpy.firstCall.args[2]).to.equal(222);
-      expect(resource1.getPendingChangeSize().height).to.equal(111);
-      expect(resource1.getPendingChangeSize().width).to.equal(222);
-    });
-
-    it('should change size when new height/width is lower', () => {
```

Comment:
From the code of the test, I suppose the old test was intended to check if `overflowCallback` not getting called when height decrease (which is a special case) but ended up with a wrong name.
I don't see why this scenario `'should change size when new height/width is lower'` is special and needed to be tested. Basically if the element is in viewport we will NOT change change size and NOT call `oveflowCallback` (what the new test cover), if the element is outside viewport, we don't care if the new height is smaller or not (we have test about element above/below viewport).